app/javascript/data.js

```diff
@@ -0,0 +1,207 @@
+import createDebug from 'debug';
+import { onLoad } from './util';
+
+const debug = createDebug('ms:data');
+
+window.store = {};
+window.results = [];
+
+/* globals store, results, ace */
+
+const addDataListRow = function () {
+  $('.data-list').prepend($('#data-list-row').clone().removeClass('template'));
+};
+
+const displaySchema = function (table) {
+  $.ajax({
```

Comment:
@ArtOfCode- now might be a good time to add https://polyfill.io — it automatically selects the necessary polyfills.